django/db/backends/base/schema.py

```diff
@@ -200,6 +200,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = timezone.now()  # default for DateTimeField
+            internal_type = field.get_internal_type()
+            if internal_type == 'DateField':
+                default = default.date()
```

Comment:
Not sure I'd consider it "broken" since no one has reported a problem. As I mentioned, it could be bad/confusing for existing users if their timestamps are suddenly shifted when they upgrade Django, but I don't care much about the issue -- just suggesting it's time wasted if we come up with better APIs (like database defaults).